day1/ques1.rb

```diff
@@ -0,0 +1,21 @@
+# 1. wap to display number is prime or not
+
+def primeOrNot(num)
+    if num<=1
+        puts  "#{num} is not prime number"
+    elsif num ==2
+        puts "#{num} is a prime number"
+    else 
+        (2...num).each do |i|
```

Comment:
Instead of iterating on all the numbers, can we reduce this iterations to some extend

Reply:
check when to use `..` and when to use `...`